package.json

```diff
@@ -174,7 +174,7 @@
     "@open-draft/logger": "^0.3.0",
     "@open-draft/until": "^2.0.0",
     "is-node-process": "^1.2.0",
-    "outvariant": "^1.2.1",
+    "outvariant": "^1.4.3",
```

Comment:
`outvariant` had issues with polymorphic errors and multiple positionals. Fixed it and updated the dependency.